sdk/logs/src/main/java/io/opentelemetry/sdk/logs/SdkLogBuilder.java

```diff
@@ -67,7 +70,11 @@ public LogBuilder setBody(String body) {
 
   @Override
   public LogBuilder setAttributes(Attributes attributes) {
```

Comment:
Arguably we should expand the `LogBuilder` interface to include setters for single typed attributes for symmetry with `SpanBuilder`. E.g. `setAttribute(String key, String value)`, `setAttribute(String key, long value)`, etc.